lib/tailwindcss/engine.rb

```diff
@@ -13,5 +13,14 @@ class Engine < ::Rails::Engine
     config.app_generators do |g|
       g.template_engine :tailwindcss
     end
+
+    server do
+      tailwind_pid = fork do
+        exec(*Tailwindcss::Commands.watch_command(always: true))
```

Comment:
Using `exec` so that `tailwindcss` executable pid stays the same as `tailwind_pid`.
If `system` is used, then line 22 `Process.kill` does not stop `tailwindcss` process when running with `puma-dev`.

Reply:
This is going to break the debugger similarly to #346 which was fixed by #349. I'll tack on a commit to fix it.